### PR TITLE
Add Fleuron for Headers

### PR DESCRIPTION
**Pull request by stephalee**
## What this does
- Adds Fleuron font files
- Styles h1 in markdown to use `Fleuron Regular` and first letter to use `Fleuron Mixed` 
- Adds a nunjucks filter that wraps each word in the first `h1` of articles in an element, with an optional classname

- [x] requires `npm install` (adds JSDOM)

## Steps to test
- try adding a class name as a parameter on [this line](https://github.com/Upstatement/ups-public-library/pull/31/files#diff-36eaee3f3f25a1ca25e89b307230f7d033e0fa17d8fa964bf62032846f4eff30R9)
- write a selector and add a style for that class

### CODE REVIEW

File: src/assets/styles/base/_markdown.scss [hunk 1]
```diff
@@ -136,7 +136,13 @@
 
   h1 {
     @include h(1);
+    font-family: $fleuron-reg;
```
Comment (stephalee) [inline src/assets/styles/base/_markdown.scss:139]: @alecmarcus is this still where we want to put type styles for entries? I'm a little unsure of where to put what/how all of the stylesheets are referencing each other

File: src/assets/styles/base/_fonts.scss [hunk 2]
```diff
@@ -29,3 +29,24 @@
   $file-name: Everett/EverettMonoTrial-Regular-web,
   $font-weight: 400
 );
+
+// ---------------------------------------------------------------------------
+// Fleuron Regular
+// ---------------------------------------------------------------------------
+
+@include font-face(
+  $family-name: FlueronRegular,
```
Comment (bchiang7) [inline src/assets/styles/base/_fonts.scss:38]: oh oops spelling error

Comment (alecmarcus) [reply src/assets/styles/base/_fonts.scss]: 😬 🙌

File: src/assets/styles/base/_markdown.scss (hunk 1 above)
Comment (alecmarcus) [reply src/assets/styles/base/_markdown.scss:139]: yes! this is a global stylesheet, for all rendered markdown across the site

File: src/assets/styles/base/_fonts.scss (hunk 2 above)
Comment (stephalee) [reply src/assets/styles/base/_fonts.scss]: That was probably... me 😅

**Review by bchiang7 — APPROVED** [review]

**Pull request merged**